### PR TITLE
improve issue template attributes

### PR DESCRIPTION
**Pull request by NotAShelf**
#### Describe your PR, what does it fix/add?

- Minor changes to the yaml syntax for the issue template.
- Example output 
- Shell renderer for the `hyprctl systeminfo` output

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)

N/A

#### Is it ready for merging, or does it need work?

Good to go.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -1,39 +1,63 @@
...
     id: ver
     attributes:
-      label: Hyprland Version
+      label: "Hyprland Version"
       description: "Paste here the output of `hyprctl version`. For hyprland after 0.34.0, paste `hyprctl systeminfo` instead."
-      value: "<details>
-        <summary>System/Version info</summary>
+      render: shell
+      placeholder: |
+        $ hyprctl systeminfo
```
Comment (vaxerski) [inline .github/ISSUE_TEMPLATE/bug.yml:21]: I don't want this in a code block. It was in a spoiler for a reason. It's long and clutters the issue.

Comment (NotAShelf) [reply .github/ISSUE_TEMPLATE/bug.yml:21]: the spoiler did not show

Comment (NotAShelf) [reply .github/ISSUE_TEMPLATE/bug.yml:21]: swappy-20240101_204216

Comment (vaxerski) [reply .github/ISSUE_TEMPLATE/bug.yml:21]: click preview idiot

Comment (NotAShelf) [reply .github/ISSUE_TEMPLATE/bug.yml:21]: it displays the text by default, who would go out of their way to click preview wtf?

Comment (vaxerski) [reply .github/ISSUE_TEMPLATE/bug.yml:21]: nobody and its not the point. The point is when they post it it will be a spoiler and wont clutter the issue.

**Pull request closed without merging**